src/StoryState.ts

```diff
@@ -274,8 +360,9 @@ export class StoryState{
 
 		copy.callStack = new CallStack(this.callStack);
 
-		copy._variablesState = new VariablesState(copy.callStack, this.story.listDefinitions);
-		copy.variablesState.CopyFrom(this.variablesState);
+		copy._variablesState = this.variablesState;
```

Comment:
I think I was just being lazy.

```csharp
internal List<string> currentErrors { get; private set; }
internal List<string> currentWarnings { get; private set; }
internal VariablesState variablesState { get; private set; }
```

is turned into

```typescript
get currentErrors(){
    return this._currentErrors;
}
private _currentErrors: string[] | null = null;

get currentWarnings(){
    return this._currentWarnings;
}
private _currentWarnings: string[] | null = null;

get variablesState(){
    return this._variablesState;
}
private _variablesState: VariablesState;
```

I guess the setters were not written back then, because they weren't required. But now, the setter for `variablesState` is needed. I'll write the `variablesState` setter instead!